pages/login.vue

```diff
@@ -60,19 +60,19 @@
                 <h2 class="flex justify-center">Analyze your listening habits</h2>
             </div>
             <div class="flex flex-col items-center">
-                <button @click="login()" class="bg-spotify-green px-8 py-4 h-min rounded-lg flex flex-row items-center font-bold">
-                    <img src="~/assets/svg/Spotify_Logo_White.svg" alt="Spotify Logo" class="h-10"/>
+                <button class="bg-spotify-green px-8 py-4 h-min rounded-lg flex flex-row items-center font-bold" @click="login()">
+                    <img src="~/assets/svg/Spotify_Logo_White.svg" alt="Spotify Logo" class="h-10">
                     <span class="pl-2 text-[25px]">login</span>
                 </button>
                 <span class="flex items-center">Secured by OAuth2.0</span>
             </div>
         </div>
         <div class="flex flex-col items-center">
             <button class="bg-purple px-4 py-2 h-min rounded-lg flex flex-row items-center font-bold">
```

Comment:
machst du gleich noch den redirect zu dem repo oder soll ich das in einer extra PR machen?

Reply:
Lieber ein extra PR